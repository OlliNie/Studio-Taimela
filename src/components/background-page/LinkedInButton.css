.social{
  font-weight: bold;
  display:flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  margin-top:2rem;
  margin-bottom:auto;
  a{
    color:white;
    padding-bottom:2rem;
  }
  p{
    color:white;
  }
}