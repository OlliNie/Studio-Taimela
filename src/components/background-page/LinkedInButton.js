import React from 'react';
import { FaLinkedin } from 'react-icons/fa';
import styles from './LinkedInButton.css';

export default function LinkedInButton() {
  return (
    <section className={styles.social}>
      <p>Find my detailed work experience on LinkedIn</p>
      <a
        alt="Linked In"
        aria-label="Linked In"
        rel="noopener noreferrer"
        target="_blank"
        href="https://www.linkedin.com/in/virpinieminen/"
      >
        <FaLinkedin size={42} />
      </a>
    </section>
  );
}
