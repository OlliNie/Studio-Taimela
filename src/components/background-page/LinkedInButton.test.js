import React from 'react';
import { shallow } from 'enzyme';
import LinkedInButton from './LinkedInButton';

describe('LinkedInButton', () => {
  it('renders LinkedInButton', () => {
    const wrapper = shallow(<LinkedInButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
