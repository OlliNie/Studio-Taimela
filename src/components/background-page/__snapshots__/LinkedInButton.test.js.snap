// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkedInButton renders LinkedInButton 1`] = `
<section
  className="social"
>
  <p>
    Find my detailed work experience on LinkedIn
  </p>
  <a
    alt="Linked In"
    aria-label="Linked In"
    href="https://www.linkedin.com/in/virpinieminen/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <FaLinkedin
      size={42}
    />
  </a>
</section>
`;
